diff --git a/vendor/cybersource/rest-client-php/lib/Authentication/Jwt/JsonWebTokenHeader.php b/vendor/cybersource/rest-client-php/lib/Authentication/Jwt/JsonWebTokenHeader.php
index 8911439a1d..6733825ebe 100644
--- a/vendor/cybersource/rest-client-php/lib/Authentication/Jwt/JsonWebTokenHeader.php
+++ b/vendor/cybersource/rest-client-php/lib/Authentication/Jwt/JsonWebTokenHeader.php
@@ -71,13 +71,19 @@ public function getJsonWebToken($jwtBody, $merchantConfig)
             throw $exception;
         }
 
-        if(!empty($cacheKey))
-            $cache_cert_store = apcu_fetch($cacheKey);
-        if($cache_cert_store ==false ){
-            $cache_cert_store="";
-            $result = apcu_store("$cacheKey", $cert_store);
-            $cache_cert_store = apcu_fetch($cacheKey);
+        if (function_exists('apcu_fetch') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN)) {
+          if(!empty($cacheKey))
+          $cache_cert_store = apcu_fetch($cacheKey);
+          if($cache_cert_store ==false ){
+              $cache_cert_store="";
+              $result = apcu_store("$cacheKey", $cert_store);
+              $cache_cert_store = apcu_fetch($cacheKey);
+          }
         }
+        else {
+          $cache_cert_store = $cert_store;
+        }
+
         //read the certificate from cert obj
         if (openssl_pkcs12_read($cache_cert_store, $cert_info, $keyPass))
         {
