diff --git a/vendor/cybersource/rest-client-php/lib/Api/KeyGenerationApi.php b/vendor/cybersource/rest-client-php/lib/Api/KeyGenerationApi.php
index 7cad7a4..5b2d837 100644
--- a/vendor/cybersource/rest-client-php/lib/Api/KeyGenerationApi.php
+++ b/vendor/cybersource/rest-client-php/lib/Api/KeyGenerationApi.php
@@ -174,7 +174,7 @@ class KeyGenerationApi
                 $printHttpBody = $httpBody;
             }
             
-            self::$logger->debug("Body Parameter :\n" . $printHttpBody); 
+            // self::$logger->debug("Body Parameter :\n" . $printHttpBody); 
         }
 
         self::$logger->debug("Return Type : \CyberSource\Model\FlexV1KeysPost200Response");
